### PR TITLE
Updates to the MRI violations test plan for 20.0 release

### PR DESCRIPTION
**Pull request by cmadjar**
This pull request updates the test plan of the MRI violations test plan. Some information in the test plan were either incorrect or misleading so changed it to make it clearer.

### CODE REVIEW

File: modules/mri_violations/test/TestPlan.md [hunk 1]
```diff
@@ -1,47 +1,64 @@
...
+1.  Check user access. Permission – `Violated Scans: View all-sites Violated 
+    Scans`
     [Automation Testing]
 2.  Check Selection Filter results for each of Patient Name, File Name,
     Series Description or Scan Type, Time Run, Type of Problem, SeriesUID
     [Manual Testing]
-3.  Filter for scans that violate MRI Protocol violation (For each type of Problem : Could not identify scan type, Visit Label does not exist, MRI Protocol Check Violation, Candidate mismatch)
-    Ensure link under Problem column goes to MRI Protocol Violations page with correct scan information 
-    set in the filters, and parameters showing when could not identify scan type, or MRI Protocol Check Violation are the type of problem
+3.  Filter for scans that violate MRI Protocol violation (For each type of
```
Comment (driusan) [inline modules/mri_violations/test/TestPlan.md]: Soon much easier to read with reasonable line lengths!

File: modules/mri_violations/test/TestPlan.md [hunk 2]
```diff
@@ -1,47 +1,64 @@
...
     [Automation Testing]
 2.  Check Selection Filter results for each of Patient Name, File Name,
     Series Description or Scan Type, Time Run, Type of Problem, SeriesUID
     [Manual Testing]
-3.  Filter for scans that violate MRI Protocol violation (For each type of Problem : Could not identify scan type, Visit Label does not exist, MRI Protocol Check Violation, Candidate mismatch)
-    Ensure link under Problem column goes to MRI Protocol Violations page with correct scan information 
-    set in the filters, and parameters showing when could not identify scan type, or MRI Protocol Check Violation are the type of problem
+3.  Filter for scans that violate MRI Protocol violation (For each type of 
+    Problem : 'Could not identify scan type', 'Visit Label does not exist',  
+    'Protocol Violation', 'CandID and PSCID doe not match' and 'CandID
```
Comment (MounaSafiHarab) [inline modules/mri_violations/test/TestPlan.md]: `do` not `doe`:)

Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: @MounaSafiHarab Thanks! Just corrected the typo.

File: modules/mri_violations/test/TestPlan.md [hunk 3]
```diff
@@ -1,47 +1,64 @@
...
-11. MRI Protocol Check violation page - click on link under PatientName column, should go to Dicom Archive for the
-    patient if permission 'Across all sites view Dicom Archive modile and pages' is granted
+11. MRI Protocol Check Violations page - click on link under PatientName 
+    column, should go to DICOM Archive for the patient if permission 'Across 
+    all sites view DICOM Archive module and pages' is granted
     [Manual Testing]
 12. Upload scan that violates protocol and ensure it shows up in the module
     [Manual Testing]
-13. Upload scan that violates protocol check and ensure it shows up in the module
+13. Upload scan that violates protocol checks and ensure it shows up in the
```
Comment (miebeers) [inline modules/mri_violations/test/TestPlan.md]: 12 and 13 are not the same thing?

File: modules/mri_violations/test/TestPlan.md [hunk 4]
```diff
@@ -1,47 +1,64 @@
...
     Series Description or Scan Type, Time Run, Type of Problem, SeriesUID
     [Manual Testing]
-3.  Filter for scans that violate MRI Protocol violation (For each type of Problem : Could not identify scan type, Visit Label does not exist, MRI Protocol Check Violation, Candidate mismatch)
-    Ensure link under Problem column goes to MRI Protocol Violations page with correct scan information 
-    set in the filters, and parameters showing when could not identify scan type, or MRI Protocol Check Violation are the type of problem
+3.  Filter for scans that violate MRI Protocol violation (For each type of 
+    Problem : 'Could not identify scan type', 'Visit Label does not exist',  
+    'Protocol Violation', 'CandID and PSCID do not match' and 'CandID 
+    does not exist').
+    Ensure that the link under the 'Problem' column bring you to:
```
Comment (miebeers) [inline modules/mri_violations/test/TestPlan.md]: brings

File: modules/mri_violations/test/TestPlan.md [hunk 5]
```diff
@@ -1,47 +1,64 @@
...
+     - the 'MRI Protocol Check Violations' page listing all violations when 
+     the link is 'Protocol Violation' (check that correct scan information 
+     are set in the filters of that page)
```
Comment (miebeers) [inline modules/mri_violations/test/TestPlan.md]: is set

File: modules/mri_violations/test/TestPlan.md [hunk 6]
```diff
@@ -1,47 +1,64 @@
...
+     - the 'MRI Protocol Violations' page listing all protocol violations 
+     with the MRI protocol parameters showing as a first table (check that 
+     correct scan information are set in the filters of that page)
```
Comment (miebeers) [inline modules/mri_violations/test/TestPlan.md]: is set

File: modules/mri_violations/test/TestPlan.md (hunk 3 above)
Comment (miebeers) [reply modules/mri_violations/test/TestPlan.md]: woops sorry, this review was supposed to have my other comments too

Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: actually, they are different. I tried rephrasing it. Hopefully it makes more sense. Those two are so confusing...

File: modules/mri_violations/test/TestPlan.md (hunk 4 above)
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md (hunk 5 above)
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md (hunk 6 above)
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md [hunk 7]
```diff
@@ -1,47 +1,65 @@
 MRI Violations Module Test Plan:
 
-1.  Check user access. Permission – `Violated Scans: View all-sites Violated Scans`
+1.  Check user access. Permission – `Violated Scans: View all-sites Violated
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md [hunk 8]
```diff
@@ -1,47 +1,65 @@
...
+    Scans`
     [Automation Testing]
 2.  Check Selection Filter results for each of Patient Name, File Name,
     Series Description or Scan Type, Time Run, Type of Problem, SeriesUID
     [Manual Testing]
-3.  Filter for scans that violate MRI Protocol violation (For each type of Problem : Could not identify scan type, Visit Label does not exist, MRI Protocol Check Violation, Candidate mismatch)
-    Ensure link under Problem column goes to MRI Protocol Violations page with correct scan information 
-    set in the filters, and parameters showing when could not identify scan type, or MRI Protocol Check Violation are the type of problem
+3.  Filter for scans that violate MRI Protocol violation (For each type of 
+    Problem : 'Could not identify scan type', 'Visit Label does not exist',
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md [hunk 9]
```diff
@@ -1,47 +1,65 @@
...
-    set in the filters, and parameters showing when could not identify scan type, or MRI Protocol Check Violation are the type of problem
+3.  Filter for scans that violate MRI Protocol violation (For each type of 
+    Problem : 'Could not identify scan type', 'Visit Label does not exist',  
+    'Protocol Violation', 'CandID and PSCID do not match' and 'CandID 
+    does not exist').
+    Ensure that the link under the 'Problem' column brings you to:
+     - the 'MRI Protocol Check Violations' page listing all violations when 
+     the link is 'Protocol Violation' (check that correct scan information 
+     is set in the filters of that page)
+     - the 'MRI Protocol Violations' page listing all protocol violations
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: Nice! Done

File: modules/mri_violations/test/TestPlan.md [hunk 10]
```diff
@@ -1,47 +1,65 @@
...
+3.  Filter for scans that violate MRI Protocol violation (For each type of 
+    Problem : 'Could not identify scan type', 'Visit Label does not exist',  
+    'Protocol Violation', 'CandID and PSCID do not match' and 'CandID 
+    does not exist').
+    Ensure that the link under the 'Problem' column brings you to:
+     - the 'MRI Protocol Check Violations' page listing all violations when 
+     the link is 'Protocol Violation' (check that correct scan information 
+     is set in the filters of that page)
+     - the 'MRI Protocol Violations' page listing all protocol violations 
+     with the MRI protocol parameters showing as a first table (check that
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: tried to clarified it.

File: modules/mri_violations/test/TestPlan.md [hunk 11]
```diff
@@ -1,47 +1,65 @@
...
+     - the 'MRI Protocol Check Violations' page listing all violations when 
+     the link is 'Protocol Violation' (check that correct scan information 
+     is set in the filters of that page)
+     - the 'MRI Protocol Violations' page listing all protocol violations 
+     with the MRI protocol parameters showing as a first table (check that 
+     correct scan information is set in the filters of that page) 
     [Manual Testing]
-4.  MRI Protocol Violations page check Selection Filter results for each of DCCID, PSCID, PatientName, SeriesUID,
-    Time Run, Series Description
+4.  MRI Protocol Violations page check Selection Filter results for each of
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md [hunk 12]
```diff
@@ -1,47 +1,65 @@
...
-    for the study
+5.  MRI Protocol Violations page - ensure first table displays all valid 
+    protocols for the study
     [Manual Testing]
 6.  MRI Protocol Violations page - displays header information of violated scan
     [Manual Testing]
-7.  MRI Protocol Violations page - allows editing of protocol table with permission 'Violated Scans: Edit MRI protocol table'.
+7.  MRI Protocol Violations page - allows editing of protocol table with 
+    permission 'Violated Scans: Edit MRI protocol table'.
     Edit values in the protocol table and verify it is saved in the database.
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md [hunk 13]
```diff
@@ -1,47 +1,65 @@
...
     [Manual Testing]
-14. Resolve issues with each of the different types of resolutions, and ensure that they show up in the resolved tab, and are no longer in the unresolved tab
+14. Resolve issues with each of the different types of resolutions, and ensure 
+    that they show up in the resolved tab, and are no longer in the unresolved
+    tab
     [Manual Testing]
 15. Ensure that the link to the protocol violation works within the resolved tab 
     [Manual Testing]
-16. Ensure all filters and sorting work in the unresolved tab
+16. Ensure all filters and sorting work in the resolved tab
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

File: modules/mri_violations/test/TestPlan.md [hunk 14]
```diff
@@ -1,47 +1,65 @@
...
     [Manual Testing]
-17. Change user access to: Take away permission for - `Violated Scans: View all-sites Violated Scans`
+17. Change user access to: Take away permission for - `Violated Scans: View
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/mri_violations/test/TestPlan.md]: done

**Pull request merged**